app/Tickets/Flipside/class.FlipsideTicketRequestEmail.php

```diff
@@ -41,7 +41,7 @@ public function getHTMLBody()
                 Reminder:  The dates for mailing your ticket request envelope are '.$this->settings['mail_start_date'].' through '
                 .$this->settings['request_stop_date'].'. Your postmark must be within that date range.<br><br>
                 If you need to update your ticket registration before mailing in your ticket request, return to 
-                <a href="https://secure.burningflipside.com/tickets/">secure.burningflipside.com/tickets/</a>.<br><br>
+                <a href="'.$this->secureUrl.'/tickets/">secure.burningflipside.com/tickets/</a>.<br><br>
```

Comment:
if or when we actually deprecate the wiki then we will deal with the requirements of deprecating the wiki.. untill that day is near we should avoid using it as a reason for anything..

if you don't like exposing the extra vars.. OK.. but the complaint against that really belongs on the pull request which actually makes that change.. and this is not that PR..

this and all it's related pull requests are seeking to decouple the hard coded urls from the system to make the dockerized dev platform function properly.. 

your pull #19 is scope creep.. plain and simple.. you are blocking my forward momentum to introduce other changes in the system which are not directly relevant to my directive..

holding up these pull requests should only be done if you have a fundamental issue with the specific changes or implementation of my pull requests.. 

if you have issues with the underlying systems or historical implementations.. those should be handled later..